ctapipe/io/containers.py

```diff
@@ -519,74 +576,73 @@ class MuonIntensityParameter(Container):
+    event_id = DeprecatedField(0, "event identification number")
+    tel_id = DeprecatedField(0, "telescope identification number")
+    ring_completeness = Field(0.0, "fraction of ring present")
+    ring_pix_completeness = Field(0.0, "fraction of pixels present in the ring")
+    ring_num_pixel = Field(0, "number of pixels in the ring image")
+    ring_size = Field(0.0, "size of the ring in pe")
+    off_ring_size = Field(0.0, "image size outside of ring in pe")
+    ring_width = Field(0.0, "width of the muon ring in degrees")
+    ring_time_width = Field(0.0, "duration of the ring image sequence")
+    impact_parameter = Field(
+        0.0, "distance of muon impact position from centre of mirror"
+    )
+    impact_parameter_chi2 = Field(0.0, "impact parameter chi squared")
+    intensity_cov_matrix = Field(0.0, "covariance matrix of intensity")
+    impact_parameter_pos_x = Field(0.0, "impact parameter x position")
+    impact_parameter_pos_y = Field(0.0, "impact parameter y position")
+    COG_x = Field(0.0, "Centre of Gravity x")
+    COG_y = Field(0.0, "Centre of Gravity y")
+    prediction = Field([], "image prediction")
+    mask = Field([], "image pixel mask")
+    optical_efficiency_muon = Field(0.0, "optical efficiency muon")
+    intensity_fit_method = Field("", "intensity fit method")
+    inputfile = Field("", "input file")
 
 
 class HillasParametersContainer(Container):
-    container_prefix = 'hillas'
+    container_prefix = "hillas"
 
-    intensity = Field(nan, 'total intensity (size)')
+    intensity = Field(nan, "total intensity (size)")
 
-    x = Field(nan, 'centroid x coordinate')
-    y = Field(nan, 'centroid x coordinate')
-    r = Field(nan, 'radial coordinate of centroid')
-    phi = Field(nan, 'polar coordinate of centroid', unit=u.deg)
+    x = Field(nan, "centroid x coordinate")
+    y = Field(nan, "centroid x coordinate")
+    r = Field(nan, "radial coordinate of centroid")
+    phi = Field(nan, "polar coordinate of centroid", unit=u.deg)
 
-    length = Field(nan, 'RMS spread along the major-axis')
-    width = Field(nan, 'RMS spread along the minor-axis')
-    psi = Field(nan, 'rotation angle of ellipse', unit=u.deg)
+    length = Field(nan, "RMS spread along the major-axis")
+    width = Field(nan, "RMS spread along the minor-axis")
+    psi = Field(nan, "rotation angle of ellipse", unit=u.deg)
 
-    skewness = Field(nan, 'measure of the asymmetry')
-    kurtosis = Field(nan, 'measure of the tailedness')
+    skewness = Field(nan, "measure of the asymmetry")
+    kurtosis = Field(nan, "measure of the tailedness")
 
 
 class LeakageContainer(Container):
     """
     Leakage
     """
-    container_prefix = ''
 
-    leakage1_pixel = Field(
-        nan,
-        'Percentage of pixels after cleaning'
-        ' that are in camera border of width=1'
+    container_prefix = "leakage"
+
+    percent_1pix = Field(
+        nan, "Percentage of pixels after cleaning that are in camera border of width=1"
```

Comment:
This is a ratio, not a percentage, right? So 10 % would be 0.1, not 10.

Reply:
good point. Should we change it to "fraction_1pix" or multiply by 100?